build/Versions.props

```diff
@@ -10,7 +10,7 @@
 
     <!-- Toolset -->
     <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha40</RoslynToolsMicrosoftRepoToolsetVersion>
-    <RoslynDependenciesOptimizationDataVersion>2.6.0-beta1-62010-04</RoslynDependenciesOptimizationDataVersion>
+    <RoslynDependenciesOptimizationDataVersion>2.6.0-beta1-62205-02</RoslynDependenciesOptimizationDataVersion>
```

Comment:
Per e-mail, if this package was produced by manually adding two extra pgos to the current roslyn optprof package, I think it would be better to just create a separate package for the project-system repo (just so we don't inadvertently drop these again with the next update).

That said, I think this is fine as an immediate fix.  👍